make-release.sh

```diff
@@ -129,9 +129,24 @@ updateVersionFile () {
   echo "${thisVERSION}" > VERSION
 }
 
+# update all registryUrl instances to the latest version
+updateRegistryUrls () {
+  thisVERSION="$1"
+  # shellcheck disable=SC2207
+  devfiles=($(find . -type f -name "devfile.yaml"))
+  baseUrl="https://eclipse-che.github.io/che-plugin-registry/"
+  for devfile in "${devfiles[@]}"
+  do
+	  sed -i "$devfile" -e "s#registryUrl: \(['\"]*\)${baseUrl}\([0-9]\+\.[0-9]\+\.[0-9]\+\)/v3/\1#registryUrl: \1${baseUrl}${thisVERSION}/v3/\1#"
```

Comment:
Seems extra complicated but I guess it's to make sure that you use `'` or `"` as defined in the file, rather than just declaring a particular quote style to use. :) Yaml is fun, no?